presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -106,6 +106,7 @@
 import io.airlift.slice.Slice;
 
 import java.util.ArrayList;
+import java.util.Comparator;
```

Comment:
This change doesn't improve `plan stability` (in plan partition columns are still in unspecified order, right?), but explain stability

Reply:
I agree that the change stabilizes the explain; Plan stability is not an issue as it relates to unspecified order of the predicates so the verbiage can be changed to use the word explain instead of plan.